Wabbajack.Lib/ModListRegistry/ModListMetadata.cs

```diff
@@ -1,18 +1,8 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
```

Comment:
I thought resharper wasn't removing these? 8P

Reply:
Lol, apparently flips a coin each time it sees it. Some still have it

Reply:
Not a huge deal since it's already done, but if there's a setting somewhere deep in there to tweak for future cleans might be nice